ledger/ledger-on-sql/src/main/scala/com/daml/ledger/on/sql/Database.scala

```diff
@@ -193,12 +185,6 @@ object Database {
   }
 
   object RDBMS {
-    object H2 extends RDBMS {
-      override val name: String = "h2"
-
-      override val queries: QueriesFactory = H2Queries.apply
```

Comment:
You should delete `H2Queries.scala` as well.

Reply:
Ugh, I was trying to rush it out. Bad Samir.